docs/src/pages/careers/accessibility-engineer.md

```diff
@@ -0,0 +1,87 @@
+- We work independently, the rest of us won't know what you're doing day-to-day unless you tell us.
+
+## About the company
+
+MUI started with Material UI, the most successful React implementation of Google's Material Design.
+It has gained a large following, not only due to the fidelity to Material Design, but also because of the number of components, its carefully designed component API, obsession for details, and community engagement.
+Today, countless teams and organizations rely on our open-source libraries to build their design system.
+
+A couple of years ago, we started to expand our suite of products.
+We released [MUI X](https://mui.com/x/), a collection of advanced components; [MUI Design kits](https://mui.com/design-kits/), the MUI components available for the most popular design tools; and also host [Templates](https://mui.com/templates/), a set of pre-built UI kits.
+We've seen rapid growth with all of them (2-3x per year) and we have more to come.
+
+We are a fully distributed team, spread across multiple time zones.
+We work mainly asynchronously, relying mostly on written collaboration.
+Every contributor has the freedom to define how they work — the rest of us won't know what you're doing day-to-day unless you tell us.
+For additional details about the MUI team and culture, you can check our [careers](https://mui.com/careers/) and [about](/about/) pages and also our [public Handbook](https://mui-org.notion.site/Handbook-f086d47e10794d5e839aef9dc67f324b).
+
+## Why we're hiring
+
+Both our open-source community and our premium products are growing fast (x2-3 YoY).
+We need talented people to keep that going!
+
+We also need help to continue to improve the health of the open-source products: make the core components easier to use, engage more with the community, polish all the details, make the components more accessible, guiding developers to answers, and just generally be a positive presence in the open-source community.
+
+### Why this is interesting
+
+Our solution empowers React developers to build awesome applications faster – we see over a million developers on MUI's docs every month.
+Hundreds of thousands of developers use MUI every month.
+
+This means that the impact of your work in improving the accessibility of our products would literally help millions of people who use the products built with our libraries.
+
+## About the role
+
+### What you'll do on a day-to-day basis
+
+Depending on the day, you'll:
+
+- **Ensure the components are accessible.**
+  You would basically be an owner of the accessibility across all MUI products. It would be your responsibility to first verify the level of accessibility the components comply with and improve it over time.
+- **Respond to issues related to accessibility.**
+  Sometimes, users of our products create issues related to a11y, it would be your responsibility to answer them, by either providing best practices or fixing issues that they may have reported.
```

Comment:
```suggestion
  Sometimes, users of our products create issues related to a11y. It would be your responsibility to answer them by either providing best practices or fixing issues that they may have reported.
```